integration_tests/_helpers.py

```diff
@@ -831,6 +837,25 @@ def _log_out(
     resp.raise_for_status()
 
 
+def _initiate_password_reset(
+    email: _Email,
+    /,
+) -> None:
+    json_ = dict(email=email)
+    resp = _httpx_client().post("auth/initiate-password-reset", json=json_)
+    resp.raise_for_status()
+
+
+def _reset_password(
+    token: _PasswordResetToken,
+    /,
+    password: _Password,
+) -> None:
+    json_ = dict(token=token, password=password)
+    resp = _httpx_client().post("auth/reset-password", json=json_)
+    resp.raise_for_status()
```

Comment:
making sure these are rate limited?

Reply:
yea, the backend puts all these under the rate limiter